test/WebSites/Basic/Controllers/FilesController.cs

```diff
@@ -27,8 +27,12 @@ public IActionResult PostFormWithFile([FromForm]FormWithFile formWithFile)
         }
 
         [HttpGet("{name}")]
-        [Produces("application/octet-stream", Type = typeof(FileResult))]
```

Comment:
Do we still want a test for `FileResult` though?

Reply:
Good question. I removed it because UseOneOfForPolymorphism is enabled and using the FileResult base type consequently produces a oneOf with four identical schemas. Testing polymorphism didn't seem the goal of this particular method and its related test cases, and the result could be confusing for others running this, so I switched it to a specific subclass instead. I'm open to switching it back, but it'll muddy things a bit and make my asserts more complicated. Just LMK what you'd prefer!

Reply:
I think it's more that people might still use the patent class like mentioned in the original issue, and I think it's better we have coverage for that as well, rather than change the existing test to be more specific to your changes.

Reply:
Done!